packages/replay/rollup.bundle.config.js

```diff
@@ -0,0 +1,36 @@
+import commonjs from '@rollup/plugin-commonjs';
+import replace from '@rollup/plugin-replace';
+
+import { makeBaseBundleConfig, makeBundleConfigVariants } from '../../rollup/index.js';
+
+import pkg from './package.json';
+
+const baseBundleConfig = makeBaseBundleConfig({
+  bundleType: 'addon',
+  entrypoints: ['src/index.ts'],
+  jsVersion: 'es6',
+  licenseTitle: '@sentry/replay',
+  outputFileBase: () => 'bundles/replay',
+  packageSpecificConfig: {
+    // We don't add @sentry/browser to the bundle, hence we mark peerDeps as external
+    external: [...Object.keys(pkg.peerDependencies || {})],
```

Comment:
I think you can remove this, as as we've learned this is merged with the base config, which already includes these anyhow, so should have no effect?

Reply:
Will check. It might be different for the bundle config though from the npm config.

Reply:
Ah, right, good point, best to double check :D (Please also make sure that actually `devDependencies` are not included there, as otherwise rrweb will again be handled weirdly)

Reply:
Ok, so for bundles, we don't mark anything as external by default. Which kinda makes sense when thinking about it 😅 
Howver, I deleted the `external` entry nevertheless because we actually don't import anything from Browser and hence rollup wouldn't bundle it anyway. This should also then work once rrweb is a dev dependency